### PR TITLE
Parse JSON after format

### PR DESCRIPTION
**Pull request by mmasaki**
This change add feature to parse JSON after format.
It would be helpful to use FluentHandler with formatters
other than FluentRecordFormatter.

In my case, I want to use FluentHnadler with oslo_log.formatters.JSONFormatter.

https://github.com/openstack/oslo.log/blob/master/oslo_log/formatters.py#L64

### CODE REVIEW

File: fluent/handler.py [hunk 1]
```diff
@@ -93,16 +93,23 @@ def __init__(self,
...
         self.tag = tag
         self.sender = sender.FluentSender(tag,
                                           host=host, port=port,
                                           timeout=timeout, verbose=verbose)
+        self.parse_json = parse_json
         logging.Handler.__init__(self)
 
     def emit(self, record):
         data = self.format(record)
+        if self.parse_json and isinstance(data, basestring):
```
Comment (repeatedly) [inline fluent/handler.py]: `basestring` is available in Python 3?

Comment (archie) [inline fluent/handler.py]: `basestring` was added in 2.3 according to https://docs.python.org/2/library/functions.html#basestring

Haven't tested this PR but would love to push support for json structures forward. Can I help somehow?

**Comment by repeatedly** [comment]
If you are still interested in this feature in this library, could you rebase the patch?
After that, I can merge this.

**Comment by arcivanov** [comment]
This has been fixed in #98

**Pull request closed without merging**